### DISCUSSION TITLE
Does Sending a Scene Off Command to a Device Already Off Cause it to Turn On Briefly?

### DISCUSSION THREAD

**Discussion by krkeegan** [TD22057/insteon-mqtt#307] [General]
This is __not__ a question about the scene commands on the modem.

The scene command on the device causes the device to send out a broadcast command to all linked devices.  I have never tried this before.  When I try and use it, if I send the scene off command to a device that is already off, it briefly causes the device to turn on before turning off.  

`./insteon-mqtt config/config.yaml scene dimmer 1 0`

Does this happen to anyone else?

**Comment by MrGibbage** [comment]
I want to make sure I am understanding the setup here. You have a switch that is a controller for a scene (device C). If a responder (device R) is already off, when device C is turned off, you are saying that you sometimes see device R turn on quickly before turning off (again)? If I have this correct, then no, I have never seen that behavior. And I have a about 40 devices and about a dozen "regular" scenes (not counting things like button parings on KPLs).

**Reply by krkeegan** [reply]
Sorry, I already answered my own question (yes it previously did do this) but I figured out why and it has already been fixed and patched.  It dealt specifically with using the scene command from the command line or `/scene` topic.  Not just a scene in general.
